Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTip.cs

```diff
@@ -14,6 +14,7 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// Creates a floating tooltip that is attached to an object and moves to stay in view as object rotates with respect to the view.
     /// </summary>
     [ExecuteAlways]
+    [DocLink("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_Tooltip.html")]
```

Comment:
I think I had this conversation at some point, but do we still prefer this custom `DocLink` attribute to Unity's built-in `HelpURL`?

https://docs.unity3d.com/ScriptReference/HelpURLAttribute.html

Reply:
Ah I didn't even know about that! I am happy to switch to that. I still want the custom documentation button as they are more obvious

Reply:
Agreed for sure about being more obvious! I don't remember the exact consensus from the previous conversation